datasheet/Zubax_GNSS2_Datasheet.tex

```diff
@@ -0,0 +1,777 @@
+%
+% Copyright (c) 2017  Zubax Robotics OU  <info@zubax.com>
+%
+% Distributed under BY-NC-ND (attribution required, non-commercial use only, no derivatives).
+%
+
+\documentclass{zubaxdoc}
+\graphicspath{{document_templates/documentation_template_latex/}}
+
+\title{Zubax GNSS 2 Datasheet}
+
+\begin{document}
+\frontmatter
+
+\begin{titlepage}
+
+\section*{Overview}
+
+Zubax GNSS 2 is a multipurpose high-performance positioning module interfaced via CAN bus, USB, and UART. It includes a high-precision barometric altimeter, a 3-axis compass with thermal compensation, and a state-of-the-art multi-system GPS / GLONASS receiver.
+\section*{Features}
+
+\begin{itemize}
+    \item State-of-the-art concurrent GPS/GLONASS receiver u-blox MAX-M8Q
+    \begin{itemize}
+    		\item Full RF shielding of the GNSS circuits ensures reliable operation in high-EMI environments
+    		\item 35 mm high gain patch antenna with large ground plane for reliable reception even in urban canyons
+    		\item Analog front end with LNA and SAW ensures high noise resilience
+    		\item Rechargeable backup battery enables low time-to-first-fix (a few seconds)
+    		\item Up to 15 Hz update rate
+    	\end{itemize}
+	\item High precision digital barometer MS5611 with  altitude resolution 10 cm
+	\item Supported interfaces:
+    \begin{itemize}
+        \item CAN, with optional double redundancy.
+        \item UART.
+        \item USB port (CDC ACM profile, also known as virtual serial port).
+    \end{itemize}
+    \item High quality assurance:
+    \begin{itemize}
+        \item Every manufactured unit undergoes a strict testing procedure.
+        The testing log for each produced unit is available to the user via the website at\\
+        \url{https://device.zubax.com/device_info}.
+        \item Protection against unlicensed (counterfeit) production by means of a digital signature
+        installed on every manufactured unit.
+    \end{itemize}
+    \item Open source firmware (3-clause BSD license).
+\end{itemize}
+
+\BeginRightColumn
+\section*{Applications}
+
+\begin{itemize}
+	\item GNSS receiver for unmanned aerial vehicles
+    \item Standalone USB GNSS receiver.
+\end{itemize}
+
+\centering
+\includegraphics[width=0.45\textwidth]{GNSS_top}
+\includegraphics[width=0.45\textwidth]{GNSS_bottom}
+
+\end{titlepage}
+
+\tableofcontents
+\BeginRightColumn
+\listoffigures
+\listoftables
+
+\mainmatter
+
+\chapter{Overview}
```

Comment:
It's a good practice to keep an empty line after a new section or new chapter declaration, let's stick to that. I'm noticing that you're sometimes being inconsistent about this: below there are places where the empty line is missing.